packages/components/message-box/src/index.vue

```diff
@@ -286,16 +285,16 @@ export default defineComponent({
       inputPattern: null,
       inputPlaceholder: '',
       inputType: 'text',
-      inputValue: null,
-      inputValidator: null,
+      inputValue: '',
+      inputValidator: undefined,
       inputErrorMessage: '',
-      message: null,
+      message: '',
       modalFade: true,
       modalClass: '',
       showCancelButton: false,
       showConfirmButton: true,
       type: '',
-      title: undefined,
+      title: '',
```

Comment:
> L39 `v-if="title !== null && title !== undefined"`

This modification will affect the previous judgment. I think it may be less error-prone to modify the type than to modify the initial value.